src/main/osd/osd_elements.c

```diff
@@ -750,6 +751,14 @@ static void osdBackgroundDisplayName(osdElementParms_t *element)
     }
 }
 
+#ifdef USE_PERSISTENT_STATS
+static void osdElementPersistentTotalFlights(osdElementParms_t *element)
+{
+    const int32_t total_flights = statsConfig()->stats_total_flights;
+    tfp_sprintf(element->buff, "%4d", total_flights);
```

Comment:
I wonder if we can find something as a symbol / name for this, or want to create one?

Reply:
@mikeller turned to be all 256 symbols are occupied and there are some discussions to refactor it to 512, but not now yet.
The closest symbol I found is a letter "T". But then it's still not very representative.

Reply:
0x12 is unused (see `drivers/osd_symbols.h` to find unused characters). 0x11 could probably be freed up by making home position related elements more consistent and switching its use to 0x05.
Or do the pragmatic thing and use `#` for 'number, count'.

Reply:
Even without extending to 512 characters (which will not be supported by all hardware) we could claim quite a few characters by doing something more efficient for the logo and reclaim the lots of empty characters that are above 0x9f at the moment. For both this and for the 512 character extension is that for this to not result in a mess, we'll have to add support for character set identification to the firmware.